### PR TITLE
ci: add check action

### PR DESCRIPTION
**Pull request by yi-Xu-0100**
### Description

Add CI to check not allowed files and lint.
ref: https://github.com/electron-vite/electron-vite-vue/pull/215#issuecomment-1194887032

See test in 
- PR: https://github.com/yi-Xu-0100/electron-vite-vue/pull/1
- Action: https://github.com/yi-Xu-0100/electron-vite-vue/actions/runs/2779851006

### What is the purpose of this pull request? 

- [ ] Bug fix
- [ ] New Feature
- [ ] Documentation update
- [x] Other

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+      - name: Comment About Changes We Can't Accept
+        if: ${{ steps.filter_not_allowed.outputs.change }}
+        uses: actions/github-script@v6
+        with:
+          script: |
+            let workflowFailMessage = "It looks like you've modified some files that we can't accept as contributions."
+            try {
+              const badFilesArr = [
+                'package-lock.json',
+                'yarn-lock.json',
```
Comment (caoxiemeihao) [inline .github/workflows/ci.yml:41]: `yarn.lock`

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+      - name: Check Not Allowed File Changes
+        uses: dorny/paths-filter@v2
+        id: filter_not_allowed
+        with:
+          list-files: json
+          filters: |
+            change:
+              - 'package-lock.json'
+              - 'yarn-lock.json'
```
Comment (caoxiemeihao) [inline .github/workflows/ci.yml:28]: `yarn.lock`

**Review by caoxiemeihao — COMMENTED** [review]
`yarn.lock`

**Review by caoxiemeihao — APPROVED** [review]
Good job! 🎉

**Comment by yi-Xu-0100** [comment]
@caoxiemeihao I updated the `reviewMessage`, and See in https://github.com/yi-Xu-0100/electron-vite-vue/pull/1#issuecomment-1202116140. The more lint needs to wait to submit next time.

If there is no other problem, you can merge it anytime. 😁

**Review by youngleish — APPROVED** [review]
thx

**Pull request merged**